elide-contrib/elide-dynamic-config-helpers/README.md

```diff
@@ -0,0 +1,19 @@
+## Validation for Dynamic Config
+
+Validate the config files in local before deployment.
+
+Sample command:
+```java
```

Comment:
lets include the mvn command to generate the jar too.

Reply:
or mention downloading the jar from maven.

Reply:
added mvn install steps